### ISSUE TITLE
space-in-parens should not throw for multiline statements

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#1351]
`space-in-parens` current reports an error on patterns like this:

``` js
  invariant(
    fromVal === toVal,
    'Unsupported transition values must be identical'
  );
```

If there is a newline before or after the paren, the rule should not warn.
